plugins/org.jkiss.dbeaver.data.transfer/src/org/jkiss/dbeaver/tools/transfer/stream/StreamTransferConsumer.java

```diff
@@ -772,19 +772,20 @@ private Path makeOutputFile(@NotNull DBRProgressMonitor monitor, @Nullable Strin
     }
 
     @NotNull
-    private Path makeOutputFile(@NotNull DBRProgressMonitor monitor, @Nullable String suffix, @NotNull String outputFolder) {
+    private Path makeOutputFile(@NotNull DBRProgressMonitor monitor, @Nullable String suffix, @NotNull String outputFolder) throws DBCException {
         Path dir;
         try {
             dir = DBFUtils.resolvePathFromString(monitor, getProject(), outputFolder);
         } catch (Exception e) {
             log.error("Error resolving output folder", e);
-            dir = Path.of(outputFolder);
+            throw new DBCException(e.getMessage(), e);
```

Comment:
What the point?
resolvePathFromString already throws DBException.

Reply:
Inside the method resolvePathFromString have use `Path.of()` that 
Throws InvalidPathException - if the path string cannot be converted to a Path